pay-api/src/pay_api/utils/converter.py

```diff
@@ -0,0 +1,23 @@
+"""Converter module to support decimal and datetime serialization."""
+from decimal import Decimal
+from datetime import datetime
+import cattrs
+
+
+class Converter(cattrs.Converter):
+    """Addon to cattr converter."""
+
+    def __init__(self):
+        """Initialize function, add in extra unstructure hooks."""
+        super().__init__()
+        # More from cattrs-extras/blob/master/src/cattrs_extras/converter.py
+        self.register_unstructure_hook(Decimal, self._unstructure_decimal)
```

Comment:
May need to extend this, I'll have to check the output versus Marshmellow for some datasets.